sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy

```diff
@@ -1326,9 +1325,107 @@ class FileAPITests extends APISpec {
         false        | false         | false     | true
     }
 
-    @Ignore
     def "Abort copy"() {
-        //TODO: Need to find a way of mocking pending copy status
+        given:
+        def fileSize = Constants.MB
+        def bytes = new byte[fileSize]
+        InputStream data = new ByteArrayInputStream(bytes)
+        ShareFileClient primaryFileClient = fileBuilderHelper(shareName, filePath).buildFileClient()
```

Comment:
Small nit to use def everywhere possible. With Groovy/Spock we have seen issues in the past where using types to define variables that it causes other variables with the same name to have the wrong type. This could have been fixed/changed in newer versions of Groovy/Spock but it's never fun to run into